Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/EnhancedSurface.azsli

```diff
@@ -24,11 +39,19 @@ class Surface
     float3 position;            //!< Position in world-space
     float3 normal;              //!< Normal in world-space
     float3 vertexNormal;        //!< Vertex normal in world-space
-    float3 albedo;              //!< Albedo color of the non-metallic material, will be multiplied against the diffuse lighting value
-    float3 specularF0;          //!< Fresnel f0 spectral value of the surface
+    float3 baseColor;           //!< Surface base color
+    float3 metallic;            //!< Surface metallic property
     float roughnessLinear;      //!< Perceptually linear roughness value authored by artists. Must be remapped to roughnessA before use
     float roughnessA;           //!< Actual roughness value ( a.k.a. "alpha roughness") to be used in microfacet calculations
     float roughnessA2;          //!< Alpha roughness ^ 2 (i.e. roughnessA * roughnessA), used in GGX, cached here for perfromance
+    float subsurfaceScatteringFactor;
+
+    //! Surface lighting inputs
+    float3 albedo;              //!< Albedo color of the non-metallic material, will be multiplied against the diffuse lighting value
```

Comment:
Why do we need baseColor and albedo?

Reply:
baseColor is referenced later in lighting and we need to decouple material srg information from the lighting phase